docs/.vitepress/theme/style.css

```
@@ -91,3 +91,139 @@
+  align-items: center;
+  gap: 0.55em;
+}
+
+.VPSidebar .VPLink .text::before,
+.VPNavBar .VPNavBarMenu a[href="https://docs.boltz.exchange"]::before {
+  content: '';
+  display: inline-block;
+  width: 1.35em;
+  height: 1.35em;
+  flex-shrink: 0;
+  background-color: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)